### DISCUSSION TITLE
web-vault image not up to date

### DISCUSSION THREAD

**Discussion by akelge** [dani-garcia/vaultwarden#3823] [Q&A]
### Subject of the issue
I just updated vaultwarden to the latest version (1.29.2-alpine) using the docker image, but I can see that web-vault is still on old version (2023.7.1) when 2023.8.2 is present.
Checking the Dockerfile I can see that web-vault image is used based on the hash, and it is still pointing to the hash of 2023.7.1.
Is this done on purpose or just an honest mistake?

I can provide a PR to upgrade web-vault too, if needed

**Comment by BlackDex** [comment]
It is on porpoise, and there already is a PR for this.
We have seen several issues with (other) v2023.8.x clients and it is better to stay on a know working version from our point of view for the web-vault in our stable version.

The only reason we needed to release a new stable was that the clients expected some new response values.
